tools/wordcloud/macros.xml

```diff
@@ -0,0 +1,30 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.9.4</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@PROFILE@">23.0</token>
+    <xml name="requirements">
+        <requirement type="package" version="1.9.4">wordcloud</requirement>
+        <requirement type="package" version="2.7.0">codecov</requirement>
+        <requirement type="package" version="7.3.2">coverage</requirement>
+        <requirement type="package" version="3.8.0">flake8</requirement>
+        <requirement type="package" version="7.4.3">pytest</requirement>
+        <requirement type="package" version="4.1.0">pytest-cov</requirement>
+        <requirement type="package" version="0.9.7">pytest-sugar</requirement>
+        <requirement type="package" version="28.0.0">setuptools</requirement>
+        <requirement type="package" version="4.0.2">twine</requirement>
+        <requirement type="package" version="0.38.1">wheel</requirement>
+        <requirement type="package" version="1.5.1">numpydoc</requirement>
+        <requirement type="package" version="2.31.1">imageio</requirement>
+        <requirement type="package" version="7.2.3">sphinx</requirement>
+        <requirement type="package" version="1.2.2">sphinx_rtd_theme</requirement>
+        <requirement type="package" version="0.8.2">sphinx_gallery</requirement>
+        <requirement type="package" version="0.4.0">sphinx-argparse</requirement>
+        <requirement type="package" version="3.0.1">sphinx-issues</requirement>
+        <requirement type="package" version="3.8.0">matplotlib</requirement>
+        <requirement type="package" version="6.0.4">multidict</requirement>
+        <requirement type="package" version="0.42.1">jieba</requirement>
+        <requirement type="package" version="1.11.3">scipy</requirement>
+        <requirement type="package" version="2.0.2">python-bidi</requirement>
+        <requirement type="package" version="3.0.0">arabic_reshaper</requirement>
```

Comment:
Why is this all needed? I would assume that only wordcloud as dependency works?

Reply:
> Why is this all needed? I would assume that only wordcloud as dependency works?

Yes, it works with only Wordclous as a requirement.